survey_compassion/models/survey_phone.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nicolas Badoux <n.badoux@hotmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+import uuid
+
+from odoo.exceptions import UserError
+from odoo import models, fields, api, _
+logger = logging.getLogger(__name__)
+
+
+class SurveyPhone(models.TransientModel):
```

Comment:
It's a good practice to match the odoo model name, you could put `SurveyMailComposeMessage`